trillian_log_api.proto

```diff
@@ -138,9 +138,30 @@ service TrillianLog {
     }
 }
 
+// Charge describes the user(s) associated with the request whose quota should
+// be checked and charged.
+message Charge {
+  // user is a list of personality-defined strings.
+  // Trillian will treat them as /User/%{user}/... keys when checking and
+  // charging quota.
+  // If one or more of the specified users has insufficient quota, the
+  // request will be denied.
+  //
+  // As an example, a Certificate Transparency frontent might set the following
+  // user strings when sending a QueueLeaves request to the Trillian log:
+  //   - the requesting IP address.
```

Comment:
(nit)  "The requesting IP address."  (capitalization is odd otherwise)

Reply:
done